### ISSUE TITLE
[Legitimate Site Blocked] www.metabase.art

### ISSUE THREAD

**Issue by hypergraphdev** [MetaMask/eth-phishing-detect#6057]
https://www.metabase.art is a new domain. why in this list

**Comment by devenblake** [comment]
Duplicate of #6003, fixed in #6035, please wait for caches to clear and it'll be unblocked.
